src/test/beast/IPEndpoint_test.cpp

```diff
@@ -116,7 +116,6 @@ class IPEndpoint_test : public unit_test::suite
       BEAST_EXPECT(v4[2]==0);
       BEAST_EXPECT(v4[3]==1);
 
-      BEAST_EXPECT((!((0xff)<<16)) == 0x00000000);
```

Comment:
The implementation of IPAddressV4 contains code like:```
            (*m_value) =
                    ( (*m_value) & (~((0xff)<<m_shift)) )
                | ((v&0xff) << m_shift);
```
It looks like this test is just a check that those types of operations will work (i.e. that `0xff` isn't 16 bits). I didn't dig into it deeply though.